vars/base.groovy

```diff
@@ -1,11 +1,11 @@
 @Library('wolox-ci')
 import com.wolox.*;
 
-def call(ProjectConfiguration projectConfig, def nextClosure) {
+def call(ProjectConfiguration projectConfig, def _, def nextClosure) {
```

Comment:
Yup... awful

Reply:
Why don't we pass a hash as the argument and access each of the components in the method. Something like:

```
def call(def args) {
   ProjectConfiguration projectConfig = args.projectConfiguration
}
```

I think that in that way this can be less awful